app/ecolabel-translator/src/App.js

```diff
@@ -9,14 +9,33 @@ function App() {
     { sender: 'Assistant', text: 'Welcome to the chat!' }
   ]);
   const [isChatActive, setIsChatActive] = useState(false);
+  const [token, setToken] = useState(1);
 
-  const handleLabelSubmit = (event) => {
+  const handleLabelSubmit = async (event) => {
     event.preventDefault();
-    setResults({
-      description: 'This is a sample eco-label description.',
-      epaRecommendation: 'EPA recommends this label for sustainable products.'
-    });
+  
+    try {
+      const response = await fetch(`http://127.0.0.1:5000/search?q=${label}`, {
```

Comment:
When I ran the site, I couldn't get it to run. There is a type error in the fetch..
`Error submitting label: TypeError: Failed to fetch
    at handleLabelSubmit (App.js:18:1)`

I verified that the url used does actually access the API, so its not a connection error. It is also not a 422, so it wasn't thrown by the api